tests/test_docs.py

```diff
@@ -45,7 +45,13 @@ def extract_code_chunks(path: Path, text: str, offset: int):
         code = m_code.group(2)
         end_line = start_line + code.count("\n") + 1
         source = "\n" * start_line + code
-        if "test='skip'" in prefix:
+        if "test='skip'" in source:
```

Comment:
Not necessary for this PR but I think it would be good to find a solution that doesn't show this comment in the documentation

Reply:
Hm ok tried a couple of things but didn't seem to work - I think test_docs can't rewrite the source?